activate.sh

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/bash
+# This script registers dot plugins so that we can use graphviz 
+# to write png images
+dot -c
```

Comment:
the `dot` tool used by graphviz uses a post-installation step to register different file format backends. this script performs that step. without this, the graphviz-related tests will fail in a fresh environment.